MapboxMaps.podspec

```diff
@@ -12,19 +12,20 @@ Pod::Spec.new do |m|
   m.author = { 'Mapbox' => 'mobile@mapbox.com' }
   m.social_media_url = 'https://twitter.com/mapbox'
   m.documentation_url = 'https://docs.mapbox.com/ios/beta/maps/api-reference/'
-
-  m.source = { http: "https://api.mapbox.com/downloads/v2/mobile-maps-ios/releases/ios/packages/#{maps_version.to_s}/MapboxMaps.xcframework.zip" }
-  m.vendored_frameworks = 'MapboxMaps.xcframework'
-
+  
+  m.source = { :git => 'https://github.com/mapbox/mapbox-maps-ios.git', :tag => 'v10.0.0-beta.13' }
   m.platform = :ios
   m.ios.deployment_target = '11.0'
   m.swift_version = '5.3'
   m.requires_arc = true
   m.pod_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64' }
 
+  m.source_files = 'Mapbox/MapboxMaps/**/*.{swift,h}', 'Mapbox/MapboxMapsAnnotations/**/*.{swift,h}', 'Mapbox/MapboxMapsFoundation/**/*.{swift,h}', 'Mapbox/MapboxMapsGestures/**/*.{swift,h}', 'Mapbox/MapboxMapsLocation/**/*.{swift,h}', 'Mapbox/MapboxMapsOffline/**/*.{swift,h}', 'Mapbox/MapboxMapsOrnaments/**/*.{swift,h,strings}', 'Mapbox/MapboxMapsSnapshot/**/*.{swift,h}', 'Mapbox/MapboxMapsStyle/**/*.{swift,h}'
+  m.resources = 'Mapbox/MapboxMapsLocation/Pucks/IndicatorAssets.xcassets'
+
   m.dependency 'MapboxCoreMaps', '10.0.0-beta.14.1'
   m.dependency 'MapboxCommon', '10.0.0-beta.9.1'
   m.dependency 'MapboxMobileEvents', '0.10.7'
-  m.dependency 'Turf', '2.0.0-alpha.2'
+  m.dependency 'Turf', '1.2.0'
```

Comment:
Since this isn’t a prerelease, please consider using the `~>` operator once again.

Reply:
Please correct me if I'm wrong but the `~>` operator is for min version right? I believe we want to version lock but I'll check with the team on this

Reply:
In CocoaPods, `~>` means:

> `~> 0.1.2` Version 0.1.2 and the versions up to 0.2, not including 0.2. This operator works based on the last component that you specify in your version requirement.

[source](https://guides.cocoapods.org/syntax/podfile.html#pod)

Since Turf follows, semver, we can therefore use `~> 1.2` as the version requirement.

Reply:
Doing so is equivalent to `from: "1.2.0"` in our Package.swift